ocs_ci/ocs/ui/pvc_ui.py

```diff
@@ -96,6 +97,81 @@ def verify_pvc_ui(self, pvc_size, access_mode, vol_mode, sc_type):
             self.check_element_text(expected_text=pvc_vol_mode_new)
             logger.info(f"Verifying volume mode : {pvc_vol_mode_new}")
 
+    def pvc_resize_ui(self, pvc_name, pvc_size, new_size, sc_type):
+        """
+        Resizing pvc via UI
+
+
+        pvc_name (str): the name of pvc
+        pvc_size (str): the size of pvc (GB)
+        new_size (int): the new size of pvc (GB)
+        sc_type (str): storage class type
+
+        """
+        self.navigate_persistentvolumeclaims_page()
+
+        logger.info("Select openshift-storage project")
+        self.do_click(self.pvc_loc["pvc_project_selector"])
+        self.do_click(self.pvc_loc["select_openshift-storage_project"])
+
+        self.do_send_keys(self.pvc_loc["search_pvc"], text=pvc_name)
+
+        logger.info(f"Go to PVC {pvc_name} Page")
+        self.do_click(self.pvc_loc[pvc_name])
+
+        if sc_type in "ocs-storagecluster-ceph-rbd":
+            helpers.create_pod()
+
+        logger.info("Checking status of Pvc")
+        self.wait_for_element(self.pvc_loc["pvc-status"], text_="Bound")
+
+        logger.info("Click on Actions")
+        self.do_click(self.pvc_loc["pvc_actions"])
+
+        logger.info("Click on Expand PVC")
+        self.do_click(self.pvc_loc["expand_pvc"])
+
+        logger.info("Clearing the size of existing pvc")
+        self.do_clear(self.pvc_loc["resize-value"])
+
+        if new_size > int(pvc_size):
```

Comment:
Ideally, the new size passed from the test case should be greater than the current size. If the check is still necessary, use assert at the beginning of this function.
```
assert new_size > int(pvc_size), (f"New size of the PVC cannot be less than existing size: {new_size}")
```

Reply:
done